pages/Smeltery.md

```diff
@@ -0,0 +1,11 @@
+# Smeltery
+The smeltery is one of the fundamental Slimefun machines, since it is used to smelt dusts into ingots and to make alloys.
+**NOTE: The smeltery requires to have a flame under it to work. While smelting items, the fire might randomly go out, meaning you will have to reignite it.**
+
+Example of its setup:
+
+![](https://i.gyazo.com/bd32cb88c0c1eb5b99f784879e97ecf8.png)
+
+### Suggestions
+* Put a piece of netherrack 2 blocks under the smeltery, so the fire does not randomly go out.
```

Comment:
"does not randomly go out"

Maybe add "without actually using it" since it will randomly go out anyway.